### ISSUE TITLE
Autocomplete is annoying

### ISSUE THREAD

**Issue by michaelst** [JakeBecker/ide-elixir#13]
To give an example, when creating a multi line map, after typing a comma it suggests a list with `do` as the first suggestion, if I push enter intending to go to the next line it will auto complete the `do` statement. I have to push esc and then enter every time I want a new line. Another annoying place has been after typing `->` or closing a function `)`

**Comment by JakeBecker** [comment]
Check that you're using the latest version of the extension. This is a dupe of https://github.com/JakeBecker/ide-elixir/issues/10 and should have been fixed in the latest release.

As in that issue, I recommend changing your key to accept autocomplete suggestions to `tab` instead of `enter`: https://github.com/atom/autocomplete-plus/issues/24 .  I've made this the default in the VS Code extension, but I haven't figured out how to do that in Atom.

Also, I highly recommend trying out the ElixirLS extension for VS Code, as it's a lot more mature. It uses `tab` as the default key, but will intelligently close `do` or `fn ->` blocks without requiring you to accept autocomplete suggestions, which (IMO) is just the right balance of manual vs. automatic.
